src/Components/SmartComponents/Modals/ColumnManagementModal.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { mountWithIntl } from '../../../Helpers/MiscHelper';
+import ColumnManagementModal from './ColumnManagementModal';
+import {
+    SYSTEM_DETAILS_HEADER
+} from '../../../Helpers/constants';
+import toJson from 'enzyme-to-json';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+let wrapper;
+beforeEach(() => {
+    wrapper = mountWithIntl(
+        <Router>
+            <ColumnManagementModal appliedColumns={SYSTEM_DETAILS_HEADER}
```

Comment:
It might be a good idea to create your own header to test all combinations of `isShownByDefault` and `isUnhidable` params